wcag-ruleset-review-process.md

```diff
@@ -1,77 +1,89 @@
+The ACT TF does this work in close collaboration with the ACT Rules Community Group through regular joint meetings. Any other groups interested to contribute rules to the WCAG ruleset should contact the co-facilitator(s) of the ACT Task Force. The ACT publication process is designed to publish high-quality ACT rules to the W3C website. 
 
-## Rule Review
+## ACT Publication steps
 
-When proposing a new rule, or an update to a rule, there are three steps to go through:
+When creating a new rule, or updating a rule, there are five steps to go through. This process involves the combined effort of the ACT Task Force, and the ACT Rules Community Group. 
 
-1. **Submitting**: The rule provider submits a rule to ACT Task Force
-2. **Surveying**: The ACT Task Force reviews the rule through a survey
-3. **Approving**: The ACT Task Force prepares a proposal for AGWG
+1. Draft proposal
+2. Call for review
+3. Call for implementation
+4. Validation
+5. Publish approval
 
-The ACT Task Force will not edit rule contents. This is to avoid creating forks, where two versions of a rule are worked on in parallel, potentially with conflicts. Where necessary, the ACT Task Force will propose changes to the rule but not put those changes in.
+### 1. Draft proposal
 
-### Submitting
+New rules, and changes to rules are made through GitHub pull requests. Any member of either ACT group can author such a pull request. A pull request must have approval from three independent reviewers before it can proceed to step 2. These reviewers can not work for the same organization, and must not have been involved in writing the pull request. 
 
-The rule provider creates an issue in the [ACT GitHub repository](https://github.com/w3c/wcag-act/issues). The issue must include the following information:
+The pull request description must include information on the type of change the pull request makes, and what happens in the "call for review" phase. There are three options:
 
-1. URL of the rule
-2. License of the rule
-3. Information about any [implementations](#implementations)
+1. New rules, and significant changes that impact multiple rules should have a **two week** call for review. 
+2. Updates to rules, and changes to definitions used in a small number of rules have a **one week** call for review.
+3. Editorial changes, including minor changes to test cases can **skip** call for review.
 
-### Surveying
+### 2. Call for review
 
-A facilitator of the ACT Task Force creates a survey for the rule, in which Task Force members are asked the following questions:
+Once a pull request has the necessary approvals, the author of the pull request sends an e-mail to the ACT Rules community group mailing list with the text "[for review]" in the subject. The e-mail must include a link to the pull request, and an indication of the duration and end date of the call for review. 
```

Comment:
Once a pull request has the necessary approvals, the author of the pull request sends an e-mail to the ACT Rules community group...
-->
Once a pull request has the necessary approvals, the author of the pull request sends an e-mail to the ACT Rules Community Group... (capitalization)

Reply:
(Just making this a little easier to pull in)

```suggestion
Once a pull request has the necessary approvals, the author of the pull request sends an e-mail to the ACT Rules Community Group mailing list with the text "[for review]" in the subject. The e-mail must include a link to the pull request, and an indication of the duration and end date of the call for review. 
```